mod_reforged/hooks/items/weapons/named/named_throwing_axe.nut

```diff
@@ -5,4 +5,13 @@
 		this.m.BaseWeaponScript = "scripts/items/weapons/throwing_axe";
 		create();
 	}
+
+	o.onEquip = function()
+	{
+		this.named_weapon.onEquip();
+
+		this.addSkill(::MSU.new("scripts/skills/actives/throw_axe", function(o) {
```

Comment:
As before, we should reduce the cost of the `throw_axe` skill if you want it to be competitive against throw javelin, instead of doing it in the weapon.